examples/cactus-example-carbon-accounting-business-logic-plugin/package.json

```diff
@@ -12,7 +12,7 @@
     "dist/*"
   ],
   "scripts": {
-    "generate-sdk": "openapi-generator generate --input-spec src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/",
+    "generate-sdk": "docker run --rm -v \"${PWD}:/local\" openapitools/openapi-generator-cli:v5.0.0-beta2 generate -i ./local/src/main/json/openapi.json -g typescript-axios -o ./local/src/main/typescript/generated/openapi/typescript-axios --reserved-words-mappings protected=protected",
```

Comment:
@jordigiam FYI: the version number might need to be updated prior to merging if the other PR from @travis-payne will get merged before this PR does.

Reply:
@travis-payne Jordi got in first so you'll have some additional fun rebasing your PR (conflicts). You'll want to end up in a state where the versions are upgraded but also Jordi's change is in there.

cc: @jordigiam